src/org/scada_lts/config/ScadaConfig.java

```diff
@@ -214,6 +218,8 @@ public Boolean getBoolean(String propertyName, boolean defaultValue) {
 				return useACL.get();
 			} else if (HTTP_RETRIVER_DO_NOT_ALLOW_ENABLE_REACTIVATION.equals(propertyName) && httpRetriverDoNotAllowEnableReactivation.isPresent()) {
 				return httpRetriverDoNotAllowEnableReactivation.get();
+			} else if (DONT_CREATE_EVETS_FOR_EMAIL_ERROR.equals(propertyName) && dontCreateEventsForEmailError.isPresent()) {
+				return dontCreateEventsForEmailError.get();
```

Comment:
Is it possible to rename this variable , it means, by just put char 'N' :
DONT_CREATE_EVETS_FOR_EMAIL_ERROR - > DONT_CREATE_EVENTS_FOR_EMAIL_ERROR